### ISSUE TITLE
Trouble Loading omnipath 

### ISSUE THREAD

**Issue by sbonner0** [saezlab/pypath#148]
Hi,

I am having some trouble being able to run the following code: 

```
from pypath import omnipath
op = omnipath.db.get_db('omnipath')
```
Which is producing the following error: 

```
Traceback (most recent call last):
  File "omnipath.py", line 12, in <module>
    op = omnipath.db.get_db('omnipath')
  File "/usr/local/lib/python3.8/site-packages/pypath-0.11.30-py3.8.egg/pypath/omnipath/app.py", line 471, in get_db
    self.ensure_dataset(dataset, ncbi_tax_id = ncbi_tax_id)
  File "/usr/local/lib/python3.8/site-packages/pypath-0.11.30-py3.8.egg/pypath/omnipath/app.py", line 142, in ensure_dataset
    self.load_dataset(dataset, ncbi_tax_id = ncbi_tax_id)
  File "/usr/local/lib/python3.8/site-packages/pypath-0.11.30-py3.8.egg/pypath/omnipath/app.py", line 367, in load_dataset
    self._add_network_df(dataset, ncbi_tax_id = ncbi_tax_id)
  File "/usr/local/lib/python3.8/site-packages/pypath-0.11.30-py3.8.egg/pypath/omnipath/app.py", line 572, in _add_network_df
    network_df = self._network_df(obj, by_source = False)
  File "/usr/local/lib/python3.8/site-packages/pypath-0.11.30-py3.8.egg/pypath/omnipath/app.py", line 553, in _network_df
    obj.make_df(**kwargs)
  File "/usr/local/lib/python3.8/site-packages/pypath-0.11.30-py3.8.egg/pypath/core/network.py", line 2100, in make_df
    'Memory usage: %s ' % common.df_memory_usage(self.df)
  File "/usr/local/lib/python3.8/site-packages/pypath-0.11.30-py3.8.egg/pypath/share/common.py", line 1676, in df_memory_usage
    counts = df._data.get_dtype_counts()
AttributeError: 'BlockManager' object has no attribute 'get_dtype_counts'
```

I am sure it is my fault, however would anyone know how to be able to fix this?

**Compute Environment:**
 - OS: MacOS 10.15.6
 - Python: 3.8.5
 - pypath: 0.11.30
- pandas: 1.1.1

**Comment by deeenes** [comment]
Hi Stephen,

Thanks for finding this, it was due to a deprecated method which got removed in `pandas 1.1`.

Best,

Denes

**Comment by sbonner0** [comment]
Hi Denes,

Thanks very much for your quick reply. Let me know if I can be of any assistance - even if it is just testing a fix.

Kind Regards,

Stephen

**Comment by deeenes** [comment]
It's already fixed in 6384c8ba, should be working if you update to the last version :)

**Comment by sbonner0** [comment]
I have tested it this morning and it works perfectly - many thanks.
